docs/install-and-run.md

```diff
@@ -3,115 +3,125 @@
-  2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
-     in your `post_install` actions to add Build Settings and Scheme changes to the **Test Target**
-     for embedding EarlGrey in your app. To use the setup gem, use `gem install earlgrey` before
-     `pod install`.
+1. In the test target's section in your `Podfile`, add EarlGrey as a dependency.
 
-  Use the patten shown in the following Podfile to setup EarlGrey with the latest CocoaPods:
+```ruby
+target TEST_TARGET do
+  project PROJECT_NAME
 
-  ```ruby
-  PROJECT_NAME = 'SampleApp'
-  TEST_TARGET = 'SampleAppTests'
-  SCHEME_FILE = 'SampleAppTests.xcscheme'
+  use_frameworks! # Required for Swift Test Targets only
+  inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+  pod 'EarlGrey'
+end
+```
+
+2. Use the `configure_for_earlgrey` method from the [EarlGrey gem](https://rubygems.org/gems/earlgrey)
+in your Podfile's `post_install` section as in below.
 
-  target TEST_TARGET do
-    project PROJECT_NAME
+```ruby
+post_install do |installer|
+  require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+  configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-    use_frameworks! # Required for Swift Test Targets only
-    inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
-    pod 'EarlGrey'
-  end
+3. Your complete Podfile when using EarlGrey with the latest CocoaPods should look like:
+
+```ruby
+PROJECT_NAME = 'SampleApp'
+TEST_TARGET = 'SampleAppTests'
+SCHEME_FILE = 'SampleAppTests.xcscheme'
+
+target TEST_TARGET do
+  project PROJECT_NAME
+
+  use_frameworks! # Required for Swift Test Targets only
+  inherit! :search_paths # Required for not double-linking libraries in the app and test targets.
+  pod 'EarlGrey'
+end
+
+post_install do |installer|
+  require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
+  configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+end
+```
 
-  post_install do |installer|
-    require 'earlgrey' # Imports the EarlGrey gem for usage in your Podfile
-    configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
-  end
-  ```
+4. Install the EarlGrey gem by doing `gem install earlgrey` and then do the `pod install`.
 
-   * Call the `configure_for_earlgrey` method for each target / Scheme.
-   * For compatibility between different versions, please peruse [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md).
-     To download a particular version of the gem, use `gem install earlgrey -v x.y.z`.
+Notes:
+* Call the `configure_for_earlgrey` method for each target / Scheme.
+* For compatibility between different versions, please peruse [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md).
```

Comment:
Yes, it will be fine. In general there's no deep meaning to the Ruby gem version numbers. In this specific case, I think version parity is more important than strictly following semver.